src/NuGet.Services.Metadata.Catalog.Monitoring/Model/PackageRegistrationIndexMetadata.cs

```diff
@@ -14,31 +14,17 @@ namespace NuGet.Services.Metadata.Catalog.Monitoring
     /// </summary>
     public class PackageRegistrationIndexMetadata : PackageRegistrationLeafMetadata
     {
-        public string Authors { get; set; }
```

Comment:
We used to grab basically every possible field from the V3 registration index, even though we were only using a small subset. Now that the monitoring job is validating against the database and not the feed, in order to continue validating against these fields, we would need to add them to the query used by Db2Catalog. If we decide we would like to add some validations against these fields, we can add them to the query and add them back here.